components/compose/awesomebar/src/main/java/mozilla/components/compose/browser/awesomebar/AwesomeBar.kt

```diff
@@ -49,11 +60,74 @@ private fun Suggestions(
     onSuggestionClicked: (AwesomeBar.Suggestion) -> Unit
 ) {
     suggestions.forEach { suggestion ->
+        Suggestion(suggestion, onSuggestionClicked)
```

Comment:
We should consider using a `LazyColumn ` instead of a Column if we expect a large number of suggestions to be shown.

Reply:
Yeah, makes sense. I'll look into this in a follow-up. I also want to implement the behavior form `BrowserAwesomebar` wehere suggestions can arrive independently and asynchronously.

Reply:
Added a commit on top to use `LazyColumn`.